libchisel/src/deployer.rs

```diff
@@ -38,6 +38,9 @@ impl<'a> Deployer<'a> {
     (local $payload_size i32)
     (set_local $size (call $getCodeSize))
 
+    ;; grow memory to fit code; assumes 1 initial page
+    (call $growToSize (get_local $size))
```

Comment:
I think instead of this we may be better off using the module builder and then we can just change `(memory 1)`. What do you think?

Reply:
That makes more sense, it'll also have less runtime cost. Will update.

Reply:
Couldn't use builder to modify existing memory. Modified the module in-place.